code-dot-org/code-dot-org
# [Sprite Lab] Remove support for title tags

Addresses tech debt created with:
- https://github.com/code-dot-org/code-dot-org/pull/45180

Related work:
- https://github.com/code-dot-org/code-dot-org/pull/45629
- https://github.com/code-dot-org/code-dot-org/pull/45614
- https://github.com/code-dot-org/code-dot-org/pull/45382

Four weeks ago we needed to address a live site issue where shared behaviors and functions were being duplicated in projects every time a user refreshed the page. The root cause of this issue was inconsistencies between blocks that use `<title/>` tags, favored by cdo Blockly, and `<field/>` tags, favored by newer version of Google Blockly. In some odd cases, such as when localizing shared behaviors and functions, our XML could get into a state where both tags were temporarily being used. In the interest of resolving this quickly for users, we introduced some [new tech debt](https://github.com/code-dot-org/code-dot-org/pull/45180#discussion_r821119964).

We have now made enough progress in moving things over to field from to title that a special case is no longer needed. 



## Testing story

Steps to test this change locally:
- Open an incognito tab
- Go to http://localhost-studio.code.org:3000/
- Change site language to non-English
- Create a new Sprite Lab project

A working Sprite Lab project is evidence that the special case being removed is no longer needed.

## Follow-up work

Recommended follow-up item:
[STAR- 2139: [Sprite Lab] Ensure shared functions never include "title" tags](https://codedotorg.atlassian.net/browse/STAR-2139)

## PR Checklist:
